lib/breaker_box.ex

```diff
@@ -68,69 +68,74 @@ defmodule BreakerBox do
   @spec register(
           breaker_name :: term,
           breaker_options :: BreakerConfiguration.t(),
-          process_name :: atom()
+          process_name :: term
         ) ::
           :ok | :reset | {:error, reason :: term}
+  def register(breaker_name, breaker_options, process_name \\ __MODULE__)
+
+  def register(_breaker_name, %BreakerConfiguration{max_failures: max_failures}, _process_name)
+      when max_failures < 5 do
```

Comment:
The only constraint on `max_failures` is that it has to be greater than 1, because it's adding 1 to the underlying 0-based failure count in the `fuse` Erlang library, and it can't trip on the 0th error. 3, for example, is perfectly fine.